sky/clouds/aws.py

```diff
@@ -98,41 +99,30 @@ def get_default_ami(cls, region_name: str, instance_type: str) -> str:
             assert len(acc) == 1, acc
             acc_name = list(acc.keys())[0]
             if acc_name == 'K80':
-                # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20211208
-                # Downgrade the AMI for K80 due as it is only compatible with
-                # NVIDIA driver lower than 470.
-                amis = {
-                    'us-east-1': 'ami-0868a20f5a3bf9702',
-                    'us-east-2': 'ami-09b8825010d4dc701',
-                    # This AMI is 20210623 as aws does not provide a newer one.
-                    'us-west-1': 'ami-0b3c34d643904a734',
-                    'us-west-2': 'ami-06b3479ab15aaeaf1',
-                }
-                assert region_name in amis, region_name
-                return amis[region_name]
-        # https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;v=3;search=:64,:Ubuntu%2020,:Deep%20Learning%20AMI%20GPU%20PyTorch # pylint: disable=line-too-long
-
-        # Commented below are newer AMIs, but as other clouds do not support
-        # torch==1.13.0+cu117 we do not use these AMIs to avoid frequent updates:
-        # Deep Learning AMI GPU PyTorch 1.12.1 (Ubuntu 20.04) 20221025
-        #   Nvidia driver: 510.47.03, CUDA Version: 11.6 (supports torch==1.13.0+cu117)
-        #     'us-east-1': 'ami-0eb1f91977a3fcc1b'
-        #     'us-east-2': 'ami-0274a6db2e19b7cc6'
-        #     'us-west-1': 'ami-0fb299af41d32cfd3'
-        #     'us-west-2': 'ami-04ba15f9bd464eb20'
-        #
-        # Current AMIs:
-        # Deep Learning AMI GPU PyTorch 1.10.0 (Ubuntu 20.04) 20220308
-        #   Nvidia driver: 510.47.03, CUDA Version: 11.6 (does not support torch==1.13.0+cu117)
-        amis = {
-            'us-east-1': 'ami-0729d913a335efca7',
-            'us-east-2': 'ami-070f4af81c19b41bf',
-            # This AMI is 20210623 as aws does not provide a newer one.
-            'us-west-1': 'ami-0b3c34d643904a734',
-            'us-west-2': 'ami-050814f384259894c',
-        }
-        assert region_name in amis, region_name
-        return amis[region_name]
+                image_id = service_catalog.get_image_id_from_tag(
+                    'sky:k80-ubuntu-2004', region_name, clouds='aws')
+                assert image_id, f'No image found for region {region_name}'
```

Comment:
It seems this is not an internal logic error but can happen due to region input?

maybe: `raise ValueError('No image found in catalog for region {region_name}. Try setting a valid image_id.')`

nit: test `is None`.